### PR TITLE
Remove custom database image

### PR DESCRIPTION
**Pull request by klochowicz**
- Makes sure the local oracle can be started properly.
- cleans up the top-level dir structure a bit

See commits for details.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -147,11 +147,10 @@ services:
 
   db:
     container_name: db
-    build:
-      context: database
-      dockerfile: Dockerfile
+    image: postgres:15-alpine
     volumes:
       - postgres:/var/lib/postgresql/data
+      - ./services/database:/docker-entrypoint-initdb.d/
```
Comment (holzeis) [inline docker-compose.yml:153]: Mounting the init script like that will not change the fact that the database has to be removed before.

Comment (klochowicz) [reply docker-compose.yml:153]: yes, but the database is removed with `just wipe` - the Docker image was not.

Comment (holzeis) [reply docker-compose.yml:153]: Doesn't just wipe rub docker-compose down -v?

Comment (klochowicz) [reply docker-compose.yml:153]: yes, it runs it, but it does not rebuild the Docker *image*, which was necessary in our old method.

Comment (klochowicz) [reply docker-compose.yml:153]: 🤔 did it work for you? or did you end up manually using `psql` to add the table? or force rebuild of the `Dockerfile` ?

Comment (holzeis) [reply docker-compose.yml:153]: hmm.. I guess i did run docker-compose up --build -d at some point.

**Review by holzeis — APPROVED** [review]
Thanks 👍

**Comment by klochowicz** [comment]
bors ping

**Comment by klochowicz** [comment]
bors was stuck, so I merged the PR without it. tests were passing.

**Pull request merged**